benches/torii.rs

```diff
@@ -1,11 +1,11 @@
 use criterion::*;
 use iroha::{
-    client::query::Query,
+    client::query::GetAccountAssets,
     config::Configuration,
     consensus::sumeragi::Sumeragi,
-    model::{block::Blockchain, tx::Transaction},
-    networking::torii::Torii,
-    storage::kura::Kura,
+    kura::Kura,
```

Comment:
might split this up into kura::BlockStore and kura::WSV (for example)

Reply:
I think we should place World State View at the top level of our API:
![image](https://user-images.githubusercontent.com/5635032/77655668-28de8e00-6f8c-11ea-830a-624098a512ee.png)
If I wrote data flows right - `clients` work directly with `Torii` and `WorldStateView`. 
`Kura` in this case is internal API.

Reply:
just to note: transaction validity depends on two types: stateless and stateful

Reply:
Yes, but I find this confusing to have to states of `validity` so based on the fact that only stateful validated transactions are signed we may combine both states into Tx::Signed - but it may be discussed later - when we will try usage of Enums for such things as Tx and Blocks.